spec/models/world_location_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+describe WorldLocation do
```

Comment:
I think this should be `type: :model` to be consistent with other model tests in this repo?